### ISSUE TITLE
Extend user functionality to include forget password reset option

### ISSUE THREAD

**Issue by shumailxyz** [v2xnetwork/typescript-express-typeorm-boilerplate#3]

**Comment by ghost** [comment]
are you still working with this project?

**Comment by ghost** [comment]
I can be helpful if you want to upgrade boilerplate.

**Comment by shumailxyz** [comment]
Hi @deeptechuz,

I am not maintaining this anymore. We had used this as boilerplate for our internal projects but ever since we found out about NestJS, we refactored everything and moved over to NestJS.

Have you looked into NestJS?

**Comment by d337** [comment]
Yes I've looked into NestJS. Do you've made boilerplate for NestJS?

**Comment by ghost** [comment]
Hey @shumailxyz I've looked into NestJS. Do you've made any boilerplate with that or using Nest CLI?

**Comment by shumailxyz** [comment]
I use Nest CLI and our boilerplate is not open source yet.

But here's a nice open source NestJS based boilerplate that you can use:
https://github.com/Vivify-Ideas/nestjs-boilerplate

**Comment by ghost** [comment]
Thanks @shumailxyz it's great.

**Comment by shumailxyz** [comment]
Nice. Btw, I also made a boilerplate a while ago for NestJS starter with TypeORM and tests:
https://github.com/shumailxyz/nestjs-typeorm-auth-starter

I just didn't refer you this because the NestJS version is not latest there. Feel free to use that if you want :)

**Comment by ghost** [comment]
Sure I will.
